packages/amazonq/src/lsp/client.ts

```diff
@@ -212,22 +218,42 @@ async function onLanguageServerReady(
         await activate(client, encryptionKey, resourcePaths.ui)
     }
 
+    // Get the provider from the manager
+    const inlineCompletionProvider = inlineManager.getInlineCompletionProvider()
+
     const refreshInterval = auth.startTokenRefreshInterval(10 * oneSecond)
 
+    // Create and start the CursorUpdateManager with the provider from InlineManager
+    const cursorUpdateManager = new CursorUpdateManager(client, inlineCompletionProvider)
```

Comment:
this needs to cursorUpdateManager.stop() per user toggling auto trigger

Reply:
actually i feel we do not necessarily need to bound this to auto trigger enablement. but anyway we're disabling it in #7553 Lei

we will reevaluate how cursorUpdateManager work once other things work fine

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/7555/commits/106f6ea8cebfcf931a8a7eff19cba2fb45cb48a2